tools/packaging/build/agent-enclave-bundle/Dockerfile

```diff
@@ -1,5 +1,12 @@
 FROM rust:1.63-bullseye as builder
 
+RUN git clone --depth 1 https://github.com/inclavare-containers/rats-tls.git && \
+    cd rats-tls && \
+    git checkout 5de6fc308f2a18f4105429451c85e6173b42ab9d && \
```

Comment:
since we clone with `--depth 1` this commit stops working when new content is merged. For example, `8cfcf9bf9f7902f85c921824c420c3c380939b06` does not work anymore.